google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddPerformanceMaxProductListingGroupTree.java

```diff
@@ -247,7 +246,7 @@ private MutateOperation createRoot() {
               .setAssetGroup(ResourceNames.assetGroup(customerId, assetGroupId))
               // Since this is the root node, do not set the ParentListingGroupFilter. For all other
               // nodes, this would refer to the parent listing group filter resource name.
-              // ParentListingGroupFilter = "<PARENT FILTER NAME>"
+              // .setParentListingGroupFilter("PARENT_FILTER_NAME")
               //
               // Unlike AddPerformanceMaxRetailCampaign, the type for the root node here must be
               // Subdivision because it will have child partitions under it.
```

Comment:
nit: "Subdivision" is in .NET because they use upper camel case. Perhaps change this to SUBDIVISION to match the Java enum convention.